content/participate/index.html.haml

```diff
@@ -0,0 +1,119 @@
+---
```

Comment:
Unfortunately not, there's not the bootstrap layout blocks available in any form for asciidoc unfortunately :(